open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive.java

```diff
@@ -145,6 +145,7 @@ public void getOriginalTypes()
         update0224Databases();
         update0512DerivedSchemaAttributes();
         update0130Projects();
```

Comment:
The other types already defined in these files need to be moved into the 2.0 archive.  I have made the changes so the new types for the integration daemon are identified with 2.2 which I think these new changes should be as well.

Reply:
If I understand correctly, the changes you're making will move these other ones already to the 2.0 archive?  I'll leave this as draft until you've submitted your changes, and merge them into this (ie. letting the merge into my fork drop those that you've already moved to 2.0)...

Reply:
Done -- have merged these changes into the latest master directly, so will close this PR as changes are now on a different branch of mine (new PR to follow)...